floor_annotation/lib/floor_annotation.dart

```diff
@@ -14,4 +14,6 @@ export 'src/on_conflict_strategy.dart';
 export 'src/primary_key.dart';
 export 'src/query.dart';
 export 'src/transaction.dart';
+export 'src/type_converter.dart';
```

Comment:
Create an issue to move all non-Flutter code into the floor package (annotations included) and move Flutter dependent code into the floor_flutter package.

Reply:
Let's see what we can do to not introduce a circular dependency.

My thoughts on re-structuring the code were:
- floor - holds annotations and needed building blocks that users either use directly or the generated code uses (e.g. `InsertionAdapter`)
  - no dependencies on other floor packages
- floor_flutter - holds everything with a direct dependency to Flutter (e.g. sqflite)
  - depends on floor
  - has dev dependency on floor_generator
- floor_generator - holds the generator that users will include as a dev dependency
  - depends on floor